modules/tensor_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -135,9 +135,9 @@ DomainIntegralAction::DomainIntegralAction(const InputParameters & params)
     mooseError("DomainIntegral error: invalid q_function_type.");
 
   if (isParamValid("crack_front_points"))
-  {
     _crack_front_points = getParam<std::vector<Point>>("crack_front_points");
-  }
+  if (isParamValid("closed_loop"))
+    _closed_loop = getParam<bool>("closed_loop");
```

Comment:
This line can just be moved to the initialization list now. Just trying to be tidy. I don't see the point in initializing to false only to change the value again. Just set it to the desired value directly.